hack/observability/alloy/kustomization.yaml

```diff
@@ -1,13 +1,16 @@
 resources:
   - ../namespace.yaml
+  - alloy-config.yaml
```

Comment:
Let's still use the other approach so we would get a new configmap on config changes + auto-recreation of the pods.

```suggestion
```

And below add

```
configMapGenerator:
- name: alloy-config
  files:
  - config.alloy
```